lib/src/test/java/io/ably/lib/test/realtime/RealtimeResumeTest.java

```diff
@@ -1070,24 +993,21 @@ public void onConnectionStateChanged(ConnectionStateChange state) {
             for (ErrorInfo resendError : resendErrors) {
                 System.out.println("presence_resume_test: error "+resendError.message);
             }
-            assertTrue(
-                "Second round of messages (queued) has errors",
-                resendErrors.length == 0
-            );
+            assertEquals("Second round of messages (queued) has errors", 0, resendErrors.length);
 
-            for (PresenceMessage presenceMessage:
-                transport.getSentPresenceMessages()) {
+            for (PresenceMessage presenceMessage: transport.getSentPresenceMessages()) {
                 System.out.println("presence_resume_test: sent message with client: "+presenceMessage.clientId +" " +
                     " action:"+presenceMessage.action);
             }
-            assertEquals("Second round of messages has incorrect size", 9, transport.getSentPresenceMessages().size());
+            assertEquals("Second round of messages has incorrect size", 6, transport.getSentPresenceMessages().size());
```

Comment:
@AndyTWF can you please go through this test once. This test is passing with expected value as `9` on `main` branch. I feel, the right value should be `6`. So, as of now it's passing with value of `6` on `no-connection-serial` branch.

Reply:
Seems there might be a bug before that got fixed while working on `no-connection-serial` spec

Reply:
Agree, it looks like it should be 6

Reply:
Thanks !